project-backend/Models/Utils/ClaimCtxTypes.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace project_backend.Models.Utils
+{
+    public static class ClaimCtxTypes
+    {
+        public const string Id = "Id";
```

Comment:
There is a way that uses extension methods, and I feel like that's the cleanest way honestly.
It gets pretty hard to manage though for large enums, for which I would suggest reflection (runtime or compile time) 
If you want to do it, then for sure go ahead, otherwise, you can add it to the backlog and let someone else do that since it's pretty hard to do. 
I would still implement the extension methods though for this, since it's not THAT big, but for bigger things it may be worth it to check out reflection

Reply:
For extension methods, you'd need to create a static class with any name, that would contain a static method which returns string and has the name you want, and gets as a parameter a "this EnumType myEnum", like this:

```C#
public static class MyEnumExtension{
   public static string toString(this ClaimTypes enum){
       return enum switch {
            sometype => 'sometype';
            othertype => 'othertype';
            _ => '';
       }
   }
}
```

Or something of the sort